api/nfd/v1alpha1/types.go

```diff
@@ -53,6 +54,22 @@ type NodeFeatureSpec struct {
 	Labels map[string]string `json:"labels"`
 }
 
+// Status of a NodeFeature object.
+type NodeFeatureStatus struct {
+	// UTC time when the NodeFeature object was last updated.
+	// +optional
+	LastAppliedAt metav1.Time `json:"lastAppliedAt,omitempty"`
+	// +optional
+	// Number of features discovered.
+	NumberOfFeatures int `json:"numberOfFeatures,omitempty"`
+	// +optional
+	// Number of errors during last feature discovery.
+	NumberOfFeatureErrors int `json:"numberOfFeatureErrors,omitempty"`
```

Comment:
this sounds more like a metric to me, I don't know what others think, I would not have this here